mod/wikirate_source/set/type/source/tabs.rb

```diff
@@ -0,0 +1,42 @@
+format :html do
+  def tab_list
+    {
+      details_tab: "#{fa_icon :info} Details",
+      metrics_tab: "#{fa_icon 'bar-chart'} #{metric_count} Metrics",
+      notes_tab: "#{fa_icon 'quote-left'} Notes",
+      download_tab: { html: download_tab_link }
+    }
+  end
+  view :details_tab do
+    tab_wrap do
+      render_core structure: "source_structure", show: "header"
```

Comment:
Does `show: "header"` do something here? 
I suggest ` tab_wrap { _render_details }`,  add view `view :details, template: :haml` and move the structure to `wikirate_source/template/set/type/source/tabs/details.haml`